x-pack/osquerybeat/ext/osquery-extension/pkg/proc/cmdline_test.go

```diff
@@ -0,0 +1,49 @@
+package proc
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"testing/fstest"
+)
+
+var cmdData = `/usr/bin/processName --config=/etc/conf/config.conf`
+
+func TestReadCmdLineFS(t *testing.T) {
+	pid := "1"
+	testCases := []struct {
+		fstest fstest.MapFS
+		assert func(string, error)
+	}{
+		{
+			fstest.MapFS{
+				"hello.txt": {
+					Data: []byte("hello, world"),
+				},
+				"proc/1/cmdline": {
+					Data: []byte(cmdData),
+				},
+				"proc/1/stat": {
+					Data: []byte("some data"),
+				},
+			},
+			func(result string, err error) {
+				assert.Nil(t, err)
+				assert.Equal(t, result, cmdData)
+			}},
+		{
+			fstest.MapFS{
+				"hello.txt": {
+					Data: []byte("hello, world"),
+				},
+			},
+			func(result string, err error) {
```

Comment:
I didn't change the original behavior here. 
But I saw that different proc functions behave differently when an error occurred.
Some of them return an empty object (for example - an empty string `"",error`), and some return `nil, error`

Is the mentioned above is the expected behavior?